templates/connect-inject-deployment.yaml

```diff
@@ -50,7 +50,7 @@ spec:
                 -tls-cert-file=/etc/connect-injector/certs/{{ .Values.connectInject.certs.certName }}
                 -tls-key-file=/etc/connect-injector/certs/{{ .Values.connectInject.certs.keyName }}
 {{- else }}
-                -tls-auto=consul-connect-injector-cfg \
+                -tls-auto={{ template "consul.fullname" . }}-connect-injector-cfg \
```

Comment:
It looks like you can use the already defined `CONSUL_FULLNAME`, like it's being used in the `-tls-auto-hosts` flag below:

${CONSUL_FULLNAME}-connect-injector-cfg